wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py

```diff
@@ -27,20 +29,21 @@ def __init__(self, v=None,
         else:
             self.TimeUnits = None
 
-    #SampleMedium = wof_base.SampleMediumTypes.NOT_RELEVANT
-    #ValueType = Column(String)
-    #IsRegular = Column(Boolean)
-    #TimeSupport = Column(Float)
-    #TimeUnitsID = Column(Integer, ForeignKey('Units.UnitsID'))
-    #DataType = Column(String)
-    #GeneralCategory = wof_base.GeneralCategoryTypes.WATER_QUALITY
+    # SampleMedium = wof_base.SampleMediumTypes.NOT_RELEVANT
+    # ValueType = Column(String)
+    # IsRegular = Column(Boolean)
+    # TimeSupport = Column(Float)
+    # TimeUnitsID = Column(Integer, ForeignKey('Units.UnitsID'))
+    # DataType = Column(String)
+    # GeneralCategory = wof_base.GeneralCategoryTypes.WATER_QUALITY
```

Comment:
Leave in the commented out lines for ValueType, IsRegular and GeneralCategory. I'm not sure they're populated elsewhere. Having them here will serve as a reminder (hopefully) to look into it later. Not a priority at this time.